Runtime/Behaviours/CinemachineVirtualCamera.cs

```diff
@@ -614,5 +615,15 @@ public override void ForceCameraPosition(Vector3 pos, Quaternion rot)
             if (m_Transitions.m_OnCameraLive != null)
                 m_Transitions.m_OnCameraLive.Invoke(this, fromCam);
         }
+        
+        /// <summary>
+        /// Returns true, when the vcam has an extension or components that require input.
+        /// </summary>
+        internal override bool RequiresUserInput()
+        {
+            var components = GetComponentPipeline();
```

Comment:
It would be more efficient to move this line after the call to base.RequiresUserInput(), because if the latter returns true this never gets used